### PR TITLE
Update mime.types: Sync with IANA as of 2021-02-27

### PR DESCRIPTION
**Pull request by xadhoom**
Recently we started to serving wasm files via phoenix, and Plug.Static set the wrong mime type for them. This update addresses it.

### CODE REVIEW

File: priv/mime.types [hunk 1]
```diff
@@ -509,7 +534,7 @@ application/vnd.antix.game-component
 application/vnd.apache.thrift.binary
 application/vnd.apache.thrift.compact
 application/vnd.apache.thrift.json
-application/vnd.api+json    json-api
+application/vnd.api+json
```
Comment (josevalim) [inline priv/mime.types:537]:
```suggestion
application/vnd.api+json    json-api
```

File: test/mime_test.exs [hunk 2]
```diff
@@ -11,7 +11,7 @@ defmodule MIMETest do
 
   test "extensions/1" do
     assert extensions("application/json") == ["json"]
-    assert extensions("application/vnd.api+json") == ["json-api"]
+    assert extensions("application/vnd.api+json") == ["json"]
```
Comment (josevalim) [inline test/mime_test.exs:14]:
```suggestion
    assert extensions("application/vnd.api+json") == ["json-api"]
```

File: priv/mime.types (hunk 1 above)
Comment (xadhoom) [reply priv/mime.types:537]: Can do, even if `json-api` is not a IANA valid type?

Comment (josevalim) [reply priv/mime.types]: Yeah, it is an addition we had.

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**